external/common/include/cycle_counter.hpp

```diff
@@ -233,7 +233,13 @@ INLINE_ELAPSED(__inline__)
 #endif
 
 /* Visual C++ (FIXME: how to detect compilation for x86-64?) */
+/* Ref: Pre-defined C/C++ Compiler Macros http://sourceforge.net/p/predef/wiki/Home/ */
+// MSVC++ 8.0 _MSC_VER = 1400
 #if _MSC_VER >= 1400 && !defined(HAVE_TICK_COUNTER)
+#if defined(__amd64__) || defined(__x86_64__) || \
```

Comment:
Actually, I think the only problem is this line:

`typedef unsigned __int64 ULONG64;`

which was being included and causing problems. I'll check a more updated version of this file.